modules/mapbox/src/deck-utils.ts

```diff
@@ -94,6 +87,29 @@ export function getDeckInstance({
   return deckInstance;
 }
 
+export function getDeckProps({gl, currProps}: {gl: WebGLRenderingContext; currProps: DeckProps}) {
```

Comment:
We could add `@luma.gl/constants` to the deps instead of an arg. Preference?

Reply:
Always import constants from `@luma.gl/constants`.